ios/sdk/module/localstorage/HippyAsyncLocalStorage.m

```diff
@@ -328,6 +350,19 @@ - (NSString *)HippyGetStorageDirectory {
     return _storageDirectory;
 }
 
+- (NSUInteger)maxReadDataSize {
+    return NSUIntegerMax;
+}
+
+- (NSUInteger)maxWriteDataSize {
+    return NSUIntegerMax;
+}
```

Comment:
'NSUIntegerMax' value is not appropriate, better to give a limited size and explain to user(add some comments).